MonoGame.Framework/Platform/Input/GamePad.Android.cs

```diff
@@ -92,18 +92,21 @@ private static GamePadCapabilities CapabilitiesOfDevice(InputDevice device)
             // this will need fixing
             capabilities.HasLeftXThumbStick = hasMap[4];
             capabilities.HasLeftYThumbStick = hasMap[4];
+            capabilities.HasLeftStickButton = hasMap[4];
+
             capabilities.HasRightXThumbStick = hasMap[5];
             capabilities.HasRightYThumbStick = hasMap[5];
+            capabilities.HasRightStickButton = hasMap[5];
 
             capabilities.HasLeftShoulderButton = hasMap[6];
             capabilities.HasRightShoulderButton = hasMap[7];
             capabilities.HasLeftTrigger = hasMap[8];
             capabilities.HasRightTrigger = hasMap[9];
 
-            capabilities.HasDPadDownButton = hasMap[10];
-            capabilities.HasDPadLeftButton = hasMap[11];
-            capabilities.HasDPadRightButton = hasMap[12];
-            capabilities.HasDPadUpButton = hasMap[13];
+            capabilities.HasDPadDownButton = true;
```

Comment:
Why are these being hardcoded as true?

Reply:
Android bug. `hasKeys` always return false for the DPads because for Android is a motion event not a keydown